downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -85,6 +85,24 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
 
 ### New features
 
+#### `ldap` package
+
+- Added support for secure LDAP (LDAPS) connections with SSL/TLS. This allows applications to securely authenticate and interact with LDAP directories using encrypted connections.
+
+  ```ballerina
+  Client ldap =  check new ({
+        port: 636,
+        hostName,
+        password,
+        domainName,
+        clientSecureSocket: {
+          cert: "tests/resources/server/certs/server.crt",
+          enable: true
+        }
+    }
+  );
+  ```
```

Comment:
Shall we add this after HTTP?
We are ordering them in alphabetical order

Reply:
Fixed in [74bf5f6](https://github.com/ballerina-platform/ballerina-dev-website/pull/9688/commits/74bf5f69e94c14efd7627da0c42f0db7338f7f36)